### ISSUE TITLE
data augmentation and weight decay

### ISSUE THREAD

**Issue by Yang-YiFan** [miaow1988/ShuffleNet_V2_pytorch_caffe#13]
Hi,
In data augmentation, do you use random crop or random resized crop?
Thanks,
Yifan

**Comment by miaow1988** [comment]
The model that I released are trained by random crop + random horizontal flip.
I've tried resized crop + RGB Jitter + image distortion + random horizontal flip, but it didn't worked very well. There might be something wrong in my data augmentation codes.

**Comment by Yang-YiFan** [comment]
Hi,  @miaow1988 
I also noticed that in your protobuf log, some module's learnable  parameter's weight decay factor was set to 0 while it's lr is non-zero. Could you elaborate on which kind of parameter needs weight decay and which do not? Thanks!

**Comment by miaow1988** [comment]
It's a standard protocol for Caffe that some lr & wd of BN & Scale layers should be set to 0. Nothing special.

**Comment by Yang-YiFan** [comment]
@miaow1988  Do you mean the scale and bias factor in BN (in Pytorch that is affine=True) should not be included in weight decay?

**Comment by miaow1988** [comment]
You got it. They should not be included.

**Comment by Yang-YiFan** [comment]
Thank you for the clarification. It helps a lot!
